docs/application/dotnet/guides/multimedia/thumbnail-images.md

```diff
@@ -27,17 +27,43 @@ using Tizen.Multimedia.Util;
 > **Note**   
 > The input media file can be common content in the device storage (internal or external) or private content in your application data.
 
-<a name="get_thumbnail"></a>
+
 ## Extracting a Thumbnail
 
 To extract a thumbnail from a file, use the `ExtractAsync()` method of the [Tizen.Multimedia.Util.ThumbnailExtractor](https://samsung.github.io/TizenFX/latest/api/Tizen.Multimedia.Util.ThumbnailExtractor.html) class:
 
 ```
-async Task Extract(string testImagePath)
+async Task ExtractAsync(string testImagePath, Size size)
+{
+    ThumbnailExtractionResult result = await ThumbnailExtractor.ExtractAsync(testImagePath, size);
+
+    Log.Info("Util", "Data size is " + result.RawData.Length);
+}
+```
+
+## Extracting a Thumbnail Synchronously
+
+To extract a thumbnail synchronously from a file, use the `Extract(string path, Size size)` or `Extract(string path)` method of the [Tizen.Multimedia.Util.ThumbnailExtractor](https://samsung.github.io/TizenFX/latest/api/Tizen.Multimedia.Util.ThumbnailExtractor.html) class:
+
+```
+void Extract(string testImagePath, Size size)
+{
+    ThumbnailExtractionResult result = ThumbnailExtractor.Extract(testImagePath, size);
+
+    Log.Info("Util", "Data size is " + result.RawData.Length);
+}
+```
+
+To extract a thumbnail synchronously from a file and save it to file, use the `Extract(string path, Size size, string resultThumbnailPath)` or `Extract(string path, string resultThumbnailPath)` method of the [Tizen.Multimedia.Util.ThumbnailExtractor](https://samsung.github.io/TizenFX/latest/api/Tizen.Multimedia.Util.ThumbnailExtractor.html) class:
+```
+void Extract(string testImagePath, string resultPath, size)
```

Comment:
Question.
In description  : Extract(string path, Size size, string resultThumbnailPath)
In sample code: Extract(string testImagePath, string resultPath, size)

The order of the parameters are different. Is it ok?

Reply:
Of course, it's not. It's bug. I'll fix it.

Reply:
I mean `void Extract(string testImagePath, Size size, string resultPath)` is just for test method so it doesn't matter.
But, the statement `ThumbnailExtractor.Extract(testImagePath, resultPath, size);` was wrong.
But, I'll change it together for consistency.

Reply:
@hsgwon thanks for your cooperation! 
@mijin85cho thanks for your valuable inputs.

Reply:
@nawab-samsung , @mijin85cho ,
Thank you for your reviews. : )